libs/@local/graph/postgres-store/src/store/postgres/query/compile.rs

```diff
@@ -1001,6 +1015,12 @@ impl<'p, 'q: 'p, R: PostgresRecord> SelectCompiler<'p, 'q, R> {
                     },
                 );
 
+                if is_outer_join_chain {
+                    join_expression.join = JoinType::LeftOuter;
+                } else if join_expression.join != JoinType::Inner {
+                    is_outer_join_chain = true;
+                }
```

Comment:
@hashdotai Are we covered with this logic? It implies that we going to change any subsequent join to be a `LEFT OUTER JOIN`, even if it would be another JOIN as defined in the relations.

Reply:
Well, it commented on this code section twice separately, I assume yes 😅